tests/shape/test-shape.lua

```diff
@@ -0,0 +1,36 @@
+-- Test if shape crash when called
+-- Also generate some SVG to be used by the documentation
+-- it also "prove" that the code examples are all working
+local cairo = require( "lgi"         ).cairo
+local shape = require( "gears.shape" )
+local filepath, svgpath = ...
+
+-- This is normal when running test, it will run twice
+if (not filepath) or (not svgpath) or key then
+    os.exit(0)
```

Comment:
Ok, I pushed one last version for today, I am exhausted and I will make even more silly mistakes if I keep rebasing this stuff. The test (should) no longer run, I removed this code.